src/modules/Elsa.Workflows.Core/Extensions/DictionaryExtensions.cs

```diff
@@ -65,6 +65,15 @@ public static class DictionaryExtensions
         dictionary.Add(key, value);
         return dictionary;
     }
+    
+    /// <summary>
+    /// Merges the specified dictionary with the other dictionary.
+    /// </summary>
+    public static void Merge(this IDictionary<string, object> dictionary, IDictionary<string, object> other)
+    {
+        foreach (var (key, value) in other)
+            dictionary[key] = value;
```

Comment:
Do we want to overwrite in case of duplicate values?
Whatever we choose for this, let's make it clear in the comments/remarks

Reply:
Yes, and good idea 👍🏻